app/src/main/java/com/github/sdp/tarjetakuna/model/Coordinates.kt

```diff
@@ -1,12 +1,14 @@
 package com.github.sdp.tarjetakuna.model
 
+import java.lang.Math.abs
+
 /**
  * Represents a pair of coordinates.
  * It is used to represent a location for a user.
  */
 data class Coordinates(
-    var latitude: Float,
-    var longitude: Float
+    var latitude: Double,
```

Comment:
Why change to Double here ?

Reply:
It is because the Location object returns doubles directly, so we might as well use double instead of float so we do not need to cast it to float.
If it causes some problems with your implementation, I can change it back to float

Reply:
Ok that's perfect in this case! let's double